catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/CollectionDAO.java

```diff
@@ -2004,4 +2010,62 @@ default boolean supportsSoftDelete() {
       return false;
     }
   }
+
+  class EntitiesCountRowMapper implements RowMapper<EntitiesCount> {
+    @Override
+    public EntitiesCount map(ResultSet rs, StatementContext ctx) throws SQLException {
+      return new EntitiesCount()
+          .withTableCount(rs.getInt("tableCount"))
+          .withTopicCount(rs.getInt("topicCount"))
+          .withDashboardCount(rs.getInt("dashboardCount"))
+          .withPipelineCount(rs.getInt("pipelineCount"))
+          .withMlmodelCount(rs.getInt("mlmodelCount"))
+          .withServicesCount(rs.getInt("servicesCount"))
+          .withUserCount(rs.getInt("userCount"))
+          .withTeamCount(rs.getInt("teamCount"));
+    }
+  }
+
+  class ServicesCountRowMapper implements RowMapper<ServicesCount> {
+    @Override
+    public ServicesCount map(ResultSet rs, StatementContext ctx) throws SQLException {
+      return new ServicesCount()
+          .withDatabaseServiceCount(rs.getInt("databaseServiceCount"))
+          .withMessagingServiceCount(rs.getInt("messagingServiceCount"))
+          .withDashboardServiceCount(rs.getInt("dashboardServiceCount"))
+          .withPipelineServiceCounte(rs.getInt("pipelineServiceCount"))
+          .withMlModelServiceCount(rs.getInt("mlModelServiceCount"));
+    }
+  }
+
+  interface UtilDAO {
+    @SqlQuery(
+        "SELECT\n"
+            + "  (SELECT COUNT(*) FROM table_entity) as tableCount, \n"
+            + "  (SELECT COUNT(*) FROM topic_entity) as topicCount,\n"
+            + "  (SELECT COUNT(*) FROM dashboard_entity) as dashboardCount,\n"
+            + "  (SELECT COUNT(*) FROM pipeline_entity) as pipelineCount, \n"
+            + "  (SELECT COUNT(*) FROM ml_model_entity) as mlmodelCount,\n"
+            + "  (SELECT \n"
+            + "  \t\t(SELECT COUNT(*) FROM database_entity) +\n"
+            + "  \t\t(SELECT COUNT(*) FROM messaging_service_entity)+\n"
+            + "  \t\t(SELECT COUNT(*) FROM dashboard_service_entity)+\n"
+            + "  \t\t(SELECT COUNT(*) FROM pipeline_service_entity)+\n"
+            + "  \t\t(SELECT COUNT(*) FROM mlmodel_service_entity)\n"
+            + "  ) as servicesCount,\n"
+            + "  (SELECT COUNT(*) FROM user_entity) as userCount,\n"
```

Comment:
It will be good to get the user count excluding bots here.

But, it will need connection aware sql to handle both MySql and Postgres - https://github.com/open-metadata/OpenMetadata/blob/main/catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/CollectionDAO.java#L1719